dashboard/app/models/script.rb

```diff
@@ -608,11 +598,37 @@ def csf_finish_url
   end
 
   def summarize
+    summarized_stages = stages.map(&:summarize)
+
+    if peer_reviews_to_complete
+      levels = []
+      peer_reviews_to_complete.times do |x|
+        levels << {
+            ids: [x],
+            position: 0,
```

Comment:
Do we use `position` anywhere when rendering progress dots?  Can it also be removed in `ScriptLevel#summarize`?